index.cjs

```diff
@@ -82,7 +88,7 @@ const getValidFiles = (modules, pattern = '*.js') => {
  * in directories by module. Module name is inferred from filename.
  *
  * @param {string[]} paths - A list of paths to parse. Note that additional files in the specified
- *	directory will be scanned (e.g. `Panels/index.js` will scan all files in `Panels`).
+ *	directory will be scanned (e.g. `Panels/index.cjs` will scan all files in `Panels`).
```

Comment:
`npm run parse` works fine, without changing `index.js` extension to `index.cjs` (`.eslintrc.cjs` also)
Is there a specific reason?

Reply:
I would get a `npm run parse` error with `enact link`, I don't know if it is a OS related bug.
![npmRunParseErrorWithEnactLink](https://user-images.githubusercontent.com/63335823/195281209-bffa8f92-11d1-4841-9e7d-21d36ef1e17f.PNG)
I checked without `enact link` and it seems it works ok. I changed the extension back to `.js`

Reply:
OK